org.kde.kblackbox.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "org.kde.kblackbox",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kblackbox",
+    "rename-icon": "kblackbox",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11", 
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
why does it need all of host?

Reply:
>  I am using only the minimal set of [permissions](https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.kblackbox.json#L10)

Basically, it's based on https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.kblackbox.json#L10

```
"finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
```

Reply:
It does not seem to need home access for anything though.